pre_commit/clientlib.py

```diff
@@ -144,6 +149,14 @@ def _entry(modname):
     )
 
 
+def warn_on_unknown_keys_at_top_level(extra, orig_keys):
```

Comment:
probably just `warn_unknown_keys` now that it's being used in other places

Reply:
cool. thanks.